contents/codes/599.md

```diff
@@ -1,15 +1,15 @@
 ---
-# This is not a status code available in the standard, but it exists
-# with significant prominence in the wild
-set: 5
-code: 599
-title: Network Connect Timeout Error
+# This is not a status code available in the standard.
```

Comment:
Although I mentioned in the other PR we probably shouldn't just permissively **add** new non-standard codes, I'm also sensitive to the fact these non-standard codes have been on the site for some time. As such, I'd really like to keep the pages intact, in case someone already links directly to and relies upon them.

Rather than commenting out the pages entirely, we can add `unlisted: true` to the front matter to hide them from the homepage (https://github.com/httpstatuses/httpstatuses/pull/11). It will still build the page, but it will leave it off the homepage for the time being.

Reply:
That's a good idea. I'll implement it the way you suggested and add to this pull request.